packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -10,13 +10,35 @@
 // @remove-on-eject-end
 'use strict';
 
+// ZEAL: add fs for ssl key and cert loading
+const fs = require('fs');
 const errorOverlayMiddleware = require('react-error-overlay/middleware');
 const noopServiceWorkerMiddleware = require('react-dev-utils/noopServiceWorkerMiddleware');
 const config = require('./webpack.config.dev');
 const paths = require('./paths');
 
-const protocol = process.env.HTTPS === 'true' ? 'https' : 'http';
 const host = process.env.HOST || '0.0.0.0';
+// ZEAL: ssl file loading
+const sslPath = function(path) {
```

Comment:
The name `sslPath` doesn't communicate to me what this is doing.  Maybe `readSslFile` or something?